snapper/Snapper.cc

```diff
@@ -655,6 +655,16 @@ namespace snapper
 	    SN_THROW(AclException());
     }
 
+    bool
+    Snapper::isQuotaSupported() const
+    {
+#ifdef ENABLE_BTRFS_QUOTA
+	const Btrfs* btrfs = dynamic_cast<const Btrfs*>(getFilesystem());
+	return btrfs != nullptr;
```

Comment:
This does not look like good C++. A virtual function in Filesystem overridden in Btrfs would be much cleaner.

Reply:
I've extended the class Filesytem to check if quota is enabled.